content/en/docs/reference/glossary/Quantity.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Quantity
+id: Qty
+date: 2023-01-24
+full_link: https://kubernetes.io/docs/tasks/configure-pod-container/assign-memory-resource/
+short_description: >
+ Quantity contains GB and GiB unit. 
```

Comment:
This is misleading. Don't mention specific prefixes.

Reply:
so i need to remove "unit' right?

Reply:
I believe the suggestion is to remove the short description at its current state as it is misleading